pr-db-mongo/dependencies

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+mongodb_base_url="https://fastdl.mongodb.org/tools/db"
+
+# Specify the directory where you want to install the tools
+install_directory="/usr/local/bin"
+# Get Distributor ID and Release number
+curl -s http://httpbin.org/get | grep "cwd" | awk -F '"' '{print $4}'
+echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/4.4 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.4.list
```

Comment:
I think we should add these to the README.md as something someone needs to do once on the machine. Otherwise, we're modifying system settings & I don't think that's something we should be doing in plugin installations.

Reply:
@kilemensi then we can remove this in install-dependencies trigger and maybe add as a pre requisite.